include/simdutf/compiler_check.h

```diff
@@ -32,4 +32,10 @@
 #error simdutf requires a compiler compliant with the C++11 standard
 #endif
 
+#if defined(SIMDUTF_CPLUSPLUS17)
+#if !__has_include(<atomic>)
```

Comment:
Ok. So if atomic is not found, then we know for sure that it cannot be threaded... correct?